Applications/ConsoleReferenceClient/Quickstarts.ReferenceClient.Config.xml

```diff
@@ -17,8 +17,10 @@
       <StorePath>%LocalApplicationData%/OPC Foundation/pki/own</StorePath>
       <SubjectName>CN=Console Reference Client, C=US, S=Arizona, O=OPC Foundation, DC=localhost</SubjectName>
     </ApplicationCertificate>
+    
+    <ApplicationCertificateTypes>Rsa,NistP256,NistP384,BrainpoolP256r1,BrainpoolP384r1</ApplicationCertificateTypes>
 
```

Comment:
Discuss how the new config layout should be, @mregen provide sample from previous discussion in .NET user group.

Reply:
Feature request was to have a `ApplicationCertificate` per ECC security profile, to be able to set different subjects